Source/Microsoft.Teams.Apps.FAQPlusPlus.Common/Models/ResponseCardPayload.cs

```diff
@@ -17,5 +21,15 @@ public class ResponseCardPayload : TeamsAdaptiveSubmitActionData
         /// Gets or sets the response given by the bot to the user.
         /// </summary>
         public string KnowledgeBaseAnswer { get; set; }
+
+        /// <summary>
+        /// Gets or sets Previous Questions.
```

Comment:
@abbodh Please add more detail on what this list is.
Also it's of type QnADTO -- do we make sure to send only the minimum data required? For example if only the question ID is needed to continue the FAQ thread, we should not send the full question and answer.

Reply:
Added more details.
We are sending question ID, previous question from QnA maker for passing back context when a follow up prompt is selected.